src/services/findElement.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * A DOM node, a selector string (which will be passed to
+ * `document.querySelector()` to find the DOM node), or a function that
+ * returns a DOM node.
+ */
+export type ElementTarget = HTMLElement | string | (() => HTMLElement);
+
+export const findElement = (elementTarget?: ElementTarget) => {
```

Comment:
I hate to go full Java on you but I'd be tempted to name this something more specific. Just glancing at the API name alone my first question would immediately be "why not just use querySelector?" So maybe something like

```suggestion
export const findElementBySelectorOrRef = (elementTarget?: ElementTarget) => {
```

Line 17 does confuse me slightly, if we're already being passed a DOM element why do we need to call this fn? 🙈 but that can probably by addressed by code comments/documentation.

Speaking of, I'll add a blocking request that we add some unit tests for this new helper! Should hopefully not be too onerous/4 it blocks max (1 for each type + where it falls through to undefined).